app/helpers/theming_helper.rb

```diff
@@ -36,4 +36,8 @@ def privpolicy_url
   def user_color
     @settings.get_value("Primary Color") || Rails.configuration.primary_color_default
   end
+
+  def maintenance_banner
```

Comment:
Don't think this check is actually needed since only admins can see this page anyways

Reply:
i think u mean the above file, right?

Reply:
Oh yeah you're right. Thanks for the quick fix :)